app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -168,3 +143,24 @@
 
   }
 }
+
+.aspect-ratio-3\:2 {
```

Comment:
This is nice. Presumably so the lazy loading doesn't cause the page to jump as the images load? Might be worth documenting that in a quick comment.

Reply:
Done 👍